tst/EnergyPlus/unit/SizingManager.unit.cc

```diff
@@ -558,3 +559,3733 @@ TEST_F(EnergyPlusFixture, SizingManager_OverrideAvgWindowInSizing)
     EXPECT_EQ(state->dataGlobal->NumOfTimeStepInHour, 1);
     EXPECT_EQ(state->dataSize->NumTimeStepsInAvg, 1);
 }
+TEST_F(EnergyPlusFixture, SizingManager_ZoneSizing_Coincident_1x)
```

Comment:
8 new zone/space sizing unit tests to test coincident vs noncoincident and multipliers. They're very similar, but I couldn't figure out a way to reset everything and use the same idf snippet and make a few small changes then re-rerun sizing.